brayns/common/camera/Camera.h

```diff
@@ -176,15 +176,15 @@ class Camera : public BaseObject
        Side)
        @param stereoMode The new stereo mode
     */
-    void setStereoMode(CameraStereoMode stereoMode)
+    void setStereoMode(StereoMode stereoMode)
```

Comment:
It looks inconsistent that you can change the stereo mode after creating the camera, but you can't change the camera type. Do you really need to change the stereo mode on this camera object? Couldn't you create a new one when the user changes the stereo mode?